src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size;
+    private float threshold;
+    private Node<K, V>[] hashMap;
+
+    public MyHashMap() {
+        hashMap = new Node[DEFAULT_INITIAL_CAPACITY];
+        threshold = DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR;
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size == threshold) {
+            resize();
+        }
+        Node<K, V> newNode = new Node<>(key, value, null);
+        Node<K, V> interationNode = hashMap[findIndex(key)];
```

Comment:
```suggestion
        Node<K, V> iterationNode = hashMap[findIndex(key)];
```
let's move `findIndex(key)` to local variable - there is no need to call it 2 times